src/common/licensing/trial_manager.js

```diff
@@ -15,7 +15,7 @@ export function useTrialStatus() {
   }
 
   const extendTrial = (days) => {
-    const newEnd = addDays(trialInfo.endsAt, days)
+    const newEnd = addDays(trialInfo.startsAt, days)
```

Comment:
You found the right line to change. That's sometimes the hardest part.

Instead of sending `trialInfo.startsAt` to `addDays`, we want it to add 30 days to today's date. So you want `Date.now()` instead